airflow/contrib/auth/backends/ldap_auth.py

```diff
@@ -175,7 +194,7 @@ def try_login(username, password):
 
         search_scope = LEVEL
         if configuration.has_option("ldap", "search_scope"):
-            search_scope = SUBTREE if configuration.get("ldap", "search_scope") == "SUBTREE" else LEVEL
+            search_scope = SUBTREE if (configuration.get("ldap", "search_scope") == "SUBTREE" else LEVEL
```

Comment:
Is this extra paren supposed to be here?

Reply:
no ill remove that